### PR TITLE
(docs): Adjust user/README.md to Help Portal structure

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Adjusted the `user/README.md` to the Help Portal structure
- Added mandatory sections
- This PR should serve as the collaboration space to create content for https://help.sap.com/docs/btp/sap-business-technology-platform-internal/wip-cloud-manager-module?state=DRAFT&version=Internal
- Stay consistent with the following order for the cloud providers: 1) AWS 2) Google Cloud 3) Azure when it comes to ordering resources, tutorials, etc.

**Related issue(s)**

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -1,29 +1,79 @@
...
-Cloud Manager can provision the following cloud resources in the underlying cloud provider subscription:
+The Cloud Manager module provides the following feature:
 
-* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster
-* Virtual Private Cloud (VPC) Peering between your project and the Kyma cluster
+* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster.
+* Virtual Private Cloud (VPC) peering between your project and the Kyma cluster.
+* Google Cloud and Amazon Web Services Redis offering.
 
 ## Architecture
```
Comment (mmitoraj) [inline docs/user/README.md:18]: Defines the basic terms, objects and entities of the service and describes their relationship to each other. A diagram is a really really nice to have :)

File: docs/user/README.md [hunk 2]
```diff
@@ -1,29 +1,79 @@
...
+
+### Cloud Manager Operator
+
+### Cloud Control Reconcilers
+
+### Cloud Resources Reconcilers
+
+### Controller Runtime Manager
+
+### Custom Manager
```
Comment (mmitoraj) [inline docs/user/README.md:45]: I made up the components' names (I was just looking at the content and the diagram) please correct my waffle.

File: docs/user/README.md [hunk 3]
```diff
@@ -1,29 +1,79 @@
...
 
 ## API / Custom Resources Definitions
```
Comment (mmitoraj) [inline docs/user/README.md:47]: I'm thinking about creating a separate document in Help Portal that can list all the Resources, such as this one: https://github.com/kyma-project/cloud-manager/blob/main/docs/user/resources/README.md

Then we could remove the long list from the main doc.

File: docs/user/README.md [hunk 4]
```diff
@@ -1,29 +1,79 @@
...
+  * [AwsVpcPeering](./resources/04-30-20-aws-vpc-peering.md)
+  * [GcpVpcPeering](./resources/04-30-30-gcp-vpc-peering.md)
+* Redis
+  * [GcpRedisInstance](./resources/04-40-10-gcp-redis-instance.md)
+  * [AwsRedisInstance](./resources/04-40-20-aws-redis-instance.md)
+  * [AzureRedisInstance](./resources/04-40-30-azure-redis-instance.md)
+
+## Resource Consumption
+
+To learn more about the resources used by the Cloud Manager module, see [Kyma Modules' Sizing](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-modules-sizing?version=Cloud).
```
[the remark this answers is not in the document]
Comment (tmilos77) [reply docs/user/README.md:75]: Cloud Manager runs in KCP and has no SKR executables, so in SKR it takes zero resources

File: docs/user/README.md [hunk 5]
```diff
@@ -1,29 +1,79 @@
...
 ## Features
 
-Cloud Manager can provision the following cloud resources in the underlying cloud provider subscription:
+The Cloud Manager module provides the following feature:
 
-* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster
-* Virtual Private Cloud (VPC) Peering between your project and the Kyma cluster
+* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster.
+* Virtual Private Cloud (VPC) peering between your project and the Kyma cluster.
+* Google Cloud and Amazon Web Services Redis offering.
```
Comment (dushanpantic) [inline docs/user/README.md:16]: It is offering Redis on Azure as well

File: docs/user/README.md (hunk 3 above)
Comment (ijovovic) [reply docs/user/README.md:47]: It makes sence. Please do so.

File: docs/user/README.md [hunk 6]
```diff
@@ -1,29 +1,53 @@
 
 # Cloud Manager Module
 
+Use the Cloud Manager module to manage infrastructure providers' resources from a Kyma cluster.
+
 ## What is Cloud Manager?
 
-Cloud Manager is a central component that manages access to additional hyperscaler resources from the Kyma Runtime cluster. Its responsibility is to bring hyperscaler products/resources into the Kyma cluster in a secure way. Once Cloud Manager as a module is added to the Kyma cluster, Cloud Manager's features give you access to the respective products and resources of the infrastructure providers.
+Cloud Manager manages access to infrastructure providers' resources and products in SAP BTP, Kyma runtime. Once you add the Cloud Manager module to your Kyma cluster, it brings the resources in a secure way.
```
Comment (mmitoraj) [inline docs/user/README.md:8]:
```suggestion
Cloud Manager manages access to the chosen (IaaS) infrastructure providers' resources and products in SAP BTP, Kyma runtime. Once you add the Cloud Manager module to your Kyma cluster, it brings the resources in a secure way.
```

File: docs/user/README.md [hunk 7]
```diff
@@ -1,29 +1,53 @@
...
 ## Features
 
-Cloud Manager can provision the following cloud resources in the underlying cloud provider subscription:
+The Cloud Manager module provides the following feature:
 
-* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster
-* Virtual Private Cloud (VPC) Peering between your project and the Kyma cluster
+* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster.
+* Virtual Private Cloud (VPC) peering between your project and the Kyma cluster.
+* Google Cloud, Amazon Web Services, and Azure Redis offering.
```
Comment (mmitoraj) [inline docs/user/README.md:16]:
```suggestion
* [NFS](link): Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster.
* [VPC peering](link): Virtual Private Cloud (VPC) peering between your Kyma and remote underlying infrastructure provider's projects.
* [Redis](link]: infrastructure provider's flavor cache that can be used in your Kyma cluster.
```

Comment (vandjelk) [reply docs/user/README.md:16]: `Kyma underlying infrastructure provider's project, account, or subscription` sounds good but not sure about `remote underlying infrastructure provider's projects`.  Perhaps `underlying` can be omitted.

**Review by tmilos77 — DISMISSED** [review]
👍

**Review by grego952 — APPROVED** [review]

**Pull request merged**